packages/remix-react/__tests__/exports-test.tsx

```diff
@@ -0,0 +1,65 @@
+import * as ReactRouterDOM from "react-router-dom";
+import * as RemixReact from "@remix-run/react";
+
+let nonReExportedKeys = new Set([
+  // Internal error used by Remix
+  "AbortedDeferredError",
+  // Remix manages the router for you, so we don't re-export these
+  "BrowserRouter",
+  "HashRouter",
+  "MemoryRouter",
+  "Router",
+  "RouterProvider",
+  "createBrowserRouter",
+  "createHashRouter",
+  "createMemoryRouter",
+  // Don't re-export unsafe APIs
+  "unstable_HistoryRouter",
+  "UNSAFE_DataRouterContext",
+  "UNSAFE_DataRouterStateContext",
+  "UNSAFE_FetchersContext",
+  "UNSAFE_LocationContext",
+  "UNSAFE_NavigationContext",
+  "UNSAFE_RouteContext",
+  "UNSAFE_ViewTransitionContext",
+  "UNSAFE_useRouteId",
+  "UNSAFE_useScrollRestoration",
+]);
+
+// Eventually we should im to get these all aligned so we can
+// `export * from react-router-dom`.  Most of the differences are Remix-specific
+// type safety, plus Link/NavLink have wrappers to support prefetching
+let modifiedExports = new Set([
+  "Await", // types
+  "Link", // remix-specific prefetching loigc
+  "NavLink", // remix-specific prefetching loigc
+  "ScrollRestoration", // remix-specific SSR restoration logic
+  "defer", // types
+  "json", // types
+  "redirect", // types
+  "redirectDocument", // types
+  "useActionData", // types
+  "useFetcher", // types
+  "useLoaderData", // types
+  "useMatches", // types
+  "useRouteLoaderData", // types
```

Comment:
Remix should have these exports but it has to maintain it's own wrappers so ensure it's exported but not identical to the RR version